app/controllers/curiosity_cards_controller.rb

```diff
@@ -4,10 +4,21 @@ class CuriosityCardsController < ApplicationController
   before_action :find_curiosity_scope_by_current_user, only: [:update, :destroy]
 
   def index
-    @curiosities = CuriosityCard.all
+    @curiosities = CuriosityCard.order('created_at DESC').page(params[:page]).per(12)
```

Comment:
You could use `.order(created_at: :desc)` instead

Reply:
this `.order(create_at: :desc) `  generate a error Dangerous query method (method whose arguments are used as raw SQL) called with non-attribute argument(s): "created_at: :desc".This method should not be called with user-provided values, such as request parameters or model attributes. Known-safe values can be passed by wrapping them in Arel.sql().